cypress/support/step-definitions/accordion-steps.js

```diff
@@ -90,21 +93,7 @@ Then('Accordion {int} row is focused', (index) => {
 });
 
 When('I focus {word} accordionRow', (position) => {
-  switch (position) {
-    case 'first':
-      accordionTitleContainerNoIFrame(FIRST_ELEMENT).first().focus({ force: true });
-      break;
-    case 'last':
-      accordionTitleContainerNoIFrame(THIRD_ELEMENT).last().focus({ force: true });
-      break;
-    default: throw new Error('Could be possible to choose only first or last');
-  }
-});
-
-When('I press keyboard downarrow key {int} times', (times) => {
```

Comment:
Why did we delete this step?

Reply:
because now we have a general step in `common-steps.js` - we can use it not only for `downarrow` but for different types of the key which we have defined in `helper.js`

```
When('I press keyboard {string} key times {int}', (key, times) => {
  for (let i = 0; i < times; i++) {
    cy.focused().trigger('keydown', keyCode(key));
  }
});